## Biomasa: Generación Energética Doméstica

#### Considerando los siguientes datos:

* Potencia útil de la caldera: 25 kW.
* Rendimiento: 90 %.
* PCI biomasa: 4100 kcal/kg.
* Días de funcionamiento anuales: 165.
* Horas diarias de funcionamiento: 7.
* Densidad de gasóleo: 0,832 kg/l.

#### A partir de los datos técnicos de una instalación térmica alimentada con biomasa en una vivienda unifamiliar, se pide:

1. Calcular la energía útil demandada por la vivienda en kWh/año.
2. Deducir la energía aportada por la biomasa.
3. Establecer el consumo anual y diario del biocombustible.
4. Determinar la energía del combustible y su consumo anual si se utilizara gasóleo C de calefacción en lugar de la biomasa, sabiendo que tiene un PCI de 9,8 kWh/kg.
5. Si el coste de la biomasa fuera de 0,051 €/kg, y el del gasóleo 0,8 €/kg, ¿cuánto se ahorraría si se empleara biomasa en lugar de gasóleo C?

### 1- Energía útil en un año:

Por 165 días de funcionamiento anual y 7 horas por día tenemos que:

In [1]:
dias = 165
horas_d = 7
horas_a = dias * horas_d
print("Horas de trabajo anual:", horas_a)

Horas de trabajo anual: 1155


Al año se trabajan 1.155 horas. Teniendo en cuenta la potencia útil de la caldera, de 25 KW, tenemos que la energía que se genera anualmente es:

In [2]:
Energ_util = 25 * horas_a
print("Energía útil por año: {} KWh".format(Energ_util))

Energía útil por año: 28875 KWh


Anualmente nuestra caldera es capaz energía en el orden de **28.875 KWh**.

### 2- Deducción de energía aportada por la biomasa

Teniendo en cuenta que la caldera tiene una eficiencia de 90% y habiendo calculado la generación útil en un año, podemos estimar la energía hipotética asumiendo una eficiencia perfecta (100%), la cuál estará directamente relacionada al combustible necesario para obtener la energía útil calculada en un principio.

In [6]:
Energ_100 = Energ_util / 0.9
print("Energía hipotética 100% por año: {:.2f} KWh".format(Energ_100))

Energía hipotética 100% por año: 32083.33 KWh


El aporte energético anual por parte de la biomasa debe estar en el orden de **32.083,33 KWh**.

### 3- Consumo anual y diario de biomasa

Sabemos que el potencial calórico inferior (PCI) de la biomasa es 4.100 Kcal/Kg y que 1 KWh equivale a 860 Kcal. La cantidad de biomasa requerida en un año será:

In [7]:
PCI_Kwh = 4100 / 860 #Kcal/Kg a KWh/Kg
Biomasa = Energ_100 / PCI_Kwh #Kg
print("Cantidad de biomasa requerida por año: {:.2f} Kg".format(Biomasa))

Cantidad de biomasa requerida por año: 6729.67 Kg


Se requieren **6729,67 kilogramos** de biomasa anualmente para generar la energía útil durante el régimen de días y horas establecido. Dado que son 165 los que trabaja la caldera, podemos deducir la masa requerida por día:

In [8]:
Biomasa_dia = Biomasa / dias
print("Cantidad de biomasa requerida diariamente: {:.2f} Kg".format(Biomasa_dia))

Cantidad de biomasa requerida diariamente: 40.79 Kg


Diariamente se requieren **40,79 Kg** de biomasa para suplir la capacidad energética de la caldera.

### 4- Energía y consumo de gasóleo C:

Vamos a abordar el escenario en el cual se utiliza el gasóleo C en lugar de biomasa. Como dato tenemos que el PCI del gasóleo es de 9,8 KWh/Kg y que la densidad de este combustible es de 0,832 Kg/l. Inicialmente calculamos la masa necesaria a partir de la energía anual demandada:

In [10]:
PCI_gasol = 9.8 #KWh/Kg
Masa_gasol = Energ_100 / PCI_gasol #Kg
print("Cantidad de gasóleo requerido por año: {:.2f} Kg".format(Masa_gasol))

Cantidad de gasóleo requerido por año: 3273.81 Kg


Vemos que requerimos 3.274 kilogramos de gasóleo para obtener la misma energía, aproximadamente 51% menos masa que la biomasa. Dicha masa en volumen lo calculamos con la densidad de gasóleo dada:

In [12]:
Vol_gasol = Masa_gasol / 0.832 #Litros
print("Volumen de gasóleo requerido por año: {:.2f} l".format(Vol_gasol))
Vol_gasol_d = Vol_gasol / dias #Litros
print("Volumen de gasóleo requerido diariamente: {:.2f} l".format(Vol_gasol_d))

Volumen de gasóleo requerido por año: 3934.87 l
Volumen de gasóleo requerido diariamente: 23.85 l


Anualmente se requieren **3.934,87 litros** de gasóleo C para suplir la demanda energética de la caldera doméstica, equivalentes a **23,85 litros** diarios.

### 5- Ahorro de uso de biomasa vs gasóleo:

Vemos que el potencial energético es considerablemente mayor para el gasóleo C que para la biomasa. Pero en costo, el gasóleo también supera ampliamente a la biomasa: 0,8 €/kg versus y 0,051 €/kg, respectivamente. Evaluamos el costo de cada escenario por separado: 100% biomasa y 100% gasóleo:

In [14]:
# COSTO ANUAL 100% BIOMASA:
Precio_biomasa = 0.051 #Euros/Kg
Costo_biomasa = Biomasa * Precio_biomasa #Euros / año
print("Costo anual de biomasa: {:.2f} euros".format(Costo_biomasa))

# COSTO ANUAL 100% GASOLEO:
Precio_gasol = 0.8 #Euros/Kg
Costo_gasol = Masa_gasol * Precio_gasol #Euros / año
print("Costo anual de gasóleo: {:.2f} euros".format(Costo_gasol))

Costo anual de biomasa: 343.21 euros
Costo anual de gasóleo: 2619.05 euros


Observamos una importante diferencia de costos entre los distintos tipos de combustible: 343 euros anuales por la biomasa versus 2.619 euros por el gasóleo C.

In [15]:
ahorro = Costo_gasol - Costo_biomasa
print("Ahorro anual por uso de biomasa: {:.2f} euros".format(ahorro))

Ahorro anual por uso de biomasa: 2275.83 euros


El ahorro por usar biomasa en vez de gasóleo será de **2275,83 euros**. Luego entrarán en consideración otros aspectos como capacidad de almacenamiento, mayormente para la biomasa.